docs/topics/gradle/gradle-compilation-and-caches.md

```diff
@@ -453,22 +447,26 @@ kotlin.build.report.output=file
+| `single_file` | Saves build reports in a format of an object to a specified local file                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| `build_scan`  | Saves build reports in the `custom values` section of the [build scan](https://scans.gradle.com/). Note that the Gradle Enterprise plugin limits the number of custom values and their length. In big projects, some values could be lost                                                                                                                                                                                                                                                               |                                                                                                                                                                                                                                                                                                                                                                                               |
+| `http`        | Posts build reports using HTTP(S). The POST method sends metrics in JSON format. You can see the current version of the sent data in the [Kotlin repository](https://github.com/JetBrains/kotlin/blob/master/libraries/tools/kotlin-gradle-plugin/src/common/kotlin/org/jetbrains/kotlin/gradle/report/data/GradleCompileStatisticsData.kt). You can find samples of HTTP endpoints in [this blog post](https://blog.jetbrains.com/kotlin/2022/06/introducing-kotlin-build-reports/#enable_build_reports) |
+| `json` | Saves build reports in JSON format to a local file. By default, it's `${project_name}-date-time-<sequence_number>.json`                                                                                                                                                                                                                                                                                                                                                       |
 
 Here's a list of available options for `kotlin.build.report`:
 
 ```none
 # Required outputs. Any combination is allowed
-kotlin.build.report.output=file,single_file,http,build_scan
+kotlin.build.report.output=file,single_file,http,build_scan,json
 
 # Mandatory if single_file output is used. Where to put reports 
 # Use instead of the deprecated `kotlin.internal.single.build.metrics.file` property
-kotlin.build.report.single_file=some_filename
+kotlin.build.report.single_file="my/directory/path"
```

Comment:
it should be a file path, not a directory. Other reports will create a new file in the specified directory, but single_file report allows to store everything into one file.